css/index.css

```diff
@@ -1,3 +1,85 @@
+*{
+    box-sizing: border-box;
+}
+
+.whole {
+    padding: 10px 3%;
+    margin: auto; 
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding:10px 20px;
+    position: fixed; 
+    top: 0;
+    width: 100%;
+    transition: 0.6s;
+    margin-bottom: 5px;
+    float: right;
+    background-color: lightgray;
+
+}
+
+header.sticky {
+    padding: 5px 10px;
+    background:rgb(3, 68, 122);
+}
+
+header.sticky ul li a {
+    color: #000 ;
+}
+header .logo {
+    cursor: pointer;
+    position: relative;
+    transition: 0.6s;
+    float: left;
+    text-decoration: none;
+    background-color: bisque;
+    padding: 5px;
+    margin: 5px;
+}
+
+.logo:hover {
+    color: #0088a9; 
+}
+
+ /* nav ul */
+header .nav_links {
+    
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style-type: none;
+    background-color: bisque;
+
+}
+
+
+header .nav_links .nav_li  {
+    
+    background-color: bisque;
```

Comment:
With CSS; styles "Cascade" from the parent selector to its children. So in this case, since the parent (`header .nav_links`) already has its background colour set to `bisque`, you don't need to also do it on the child, since it will automatically inherit it `header .nav_links .nav_li` unless you expect to later on possibly change just the child `li` style independently